DependencyInjection/Configuration.php

```diff
@@ -105,6 +105,7 @@ public function getConfigTreeBuilder()
                         ->scalarNode('default_format')->defaultNull()->end()
                         ->scalarNode('prefix_methods')->defaultTrue()->end()
                         ->scalarNode('include_format')->defaultTrue()->end()
+                        ->scalarNode('routing_ignore_classes')->defaultValue([])->end()
```

Comment:
I add it but it's a feature only for php7 : https://secure.php.net/manual/fr/reflectionparameter.gettype.php